river/render.zig

```diff
@@ -88,7 +88,9 @@ pub fn renderOutput(output: *Output) void {
             if (view.focused) continue;
 
             renderView(output.*, view, &now);
-            renderBorders(output.*, view, &now);
+            if (view.draw_borders) {
+                renderBorders(output.*, view, &now);
+            }
```

Comment:
I'd personally put this on one line, feel free to ignore this one though, it's pretty arbitrary.
```suggestion
            if (view.draw_borders) renderBorders(output.*, view, &now);
```